Jenkinsfile

```diff
@@ -46,12 +52,7 @@ pipeline {
           def buildArgs = "--build-arg APP_ENV --build-arg COMMIT_ID --build-arg CONTENTFUL_ACCESS_TOKEN --build-arg CONTENTFUL_SPACE_ID --build-arg CONTENT_ASSET_PREFIX --build-arg SENTRY_CONTENT_DSN --build-arg PROJECT_ASSET_PREFIX --build-arg SENTRY_PROJECT_DSN ."
           def newImage = docker.build(dockerImageName, buildArgs)
           newImage.push()
-
-          if (BRANCH_NAME == 'master') {
-            stage('Update latest tag') {
-              newImage.push('latest')
-            }
-          }
+          newImage.push('latest')
```

Comment:
I think you still want the `branch == 'master'` conditional here to ensure the production-release tag builds don't clobber the latest master changes.

Reply:
Good point. I was assuming that master branch changes always build the `-staging` image, whereas on the tag change we want to build and push the `-production` image. Maybe I should split those builds up into two separate scripts to clarify what's going on, rather than having on script that builds both images. The code would be more verbose, but I think might be easier to read.

Reply:
I think I will do that. It would mean writing out the staging and production build environments in separate blocks, which would also make the code easier to read.

Reply:
ensure only master branch pushes to latest tag
```suggestion
```

Reply:
Don't we want to push `zooniverse/front-end-monorepo-staging:latest`and `zooniverse/front-end-monorepo-production:latest` separately?

Reply:
AH 🤦 - the image labels are different my apologies. Ignore my comment.

Reply:
No problem. The builds are slightly different because of the different asset prefixes and sentry environments.